### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by hugh-burgess**
Ready to merge, please check my progress. Thanks

### CODE REVIEW

**Review by FriederikeM — APPROVED** [review]
ok cool, bit fast on the CSS, but i'll allow it.

**Comment by joerghebestreit** [comment]
you doing great!!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <meta charset="UTF-8" />
+    <meta name="website" content="NF website Hugh" />
+    <title>My First NF Website</title>
+
+    <!--   <link href="./style.css" rel="stylesheet" /> -->
+  </head>
+
+  <body>
+    <header>
+      <h1>Hugh Burgess</h1>
```
Comment (marwinburesch) [inline index.html:13]: with two `s`, perfect!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quo, saepe
+          aperiam alias vitae quod autem iusto untur itaque in vel quidem dicta
+          temporibus cupiditate voluptatum numquam velit perferendis maiores
+          animi!
+        </p>
+      </article>
+    </section>
+
+    <section>
+      <div class="box">
```
Comment (marwinburesch) [inline index.html:39]: 👀  a box, huh?
This could be a bit more descriptive

File: index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <p>Mobile: 07234823492342213434</p>
+        <p>Email: <a href="">hello@me.com</a></p>
+      </div>
+    </section>
+
+    <article style="text-align: center" class="links" id="Contact">
```
Comment (marwinburesch) [inline index.html:45]: I "like" that you have styles in a separate file, so I think you should have a class for `text-align: center
` as well

Also I think you're missing a headline, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#usage_notes

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Good work, mate! 🐜

File: index.html (hunk 1 above)
Comment (hugh-burgess) [reply index.html:13]: ah with my name?

File: index.html (hunk 2 above)
Comment (hugh-burgess) [reply index.html:39]: done

**Pull request merged**